test/OpenSpaceToolkit/Astrodynamics/NumericalSolver.test.cpp

```diff
@@ -98,6 +130,22 @@ class OpenSpaceToolkit_Astrodynamics_NumericalSolver : public ::testing::Test
     }
 };
 
+class OpenSpaceToolkit_Astrodynamics_NumericalSolver_Parametrized
+    : public OpenSpaceToolkit_Astrodynamics_NumericalSolver,
+      public ::testing::WithParamInterface<Tuple<NumericalSolver::StepperType>>
```

Comment:
```suggestion
      public ::testing::WithParamInterface<Tuple<NumericalSolver::StepperType, Real>>
```
@kyle-cochran over here